### DISCUSSION TITLE
Cron Scheduler 3.4.7 missing readiness probe

### DISCUSSION THREAD

**Discussion by zachhutchins** [mia-platform/community#380] [Q&A] [answered]
### Which section?

Marketplace

### Which is the version?

v12.0.0

### Description

Attempting to use the cron scheduler v3.4.7. Went through the plugin template and set up to deploy I noticed the container does not have anywhere its listening for a readiness prob. The UI does not offer any alternative outside using http to test for readiness. Is this by design? I would expect to see nginx or even a background express server but not seeing anything I can use.

Using version Cron Schedule 

### Actual Outcome

Readiness probe fails leaving the pod in a bad state, although the pod does not need traffic its not good form to leave something in a non ready state.

### Expected Outcome

http server that provides /ready to establish readiness as it's the only option the UI offers.

**Comment by fredmaggiowski** [comment, marked as answer]
Hi @zachhutchins, thanks for reaching out.

Probe management has been [improved with v12.0.2](https://docs.mia-platform.eu/docs/release-notes/v12.0.2#probe-configuration-improvement) to give support to startup probe, exec probes and disabling probes altogether.

As for the chron scheduler, it is not meant to have an http probe so there is definitely an issue with that component configuration.

I'd advise you to upgrade your Console and disable the probes.
